fbw-a380x/src/systems/instruments/src/RMP/Components/RefImages.tsx

```diff
@@ -16,7 +16,7 @@ export class RefImages extends DisplayComponent<ComponentProps> {
 
   private readonly buttonRef = FSComponent.createRef<HTMLDivElement>();
 
-  onAfterRender(node: VNode): void {
+  onAfterRender(): void {
```

Comment:
Hmm, this is a negative change IMO. I wonder if we add node to the regex if it would leave it.